dist/boards/slstk3400a/doc.txt

```diff
@@ -85,6 +93,7 @@ PIN 1 is the bottom-left contact when the header faces  you horizontally.
 |                               | Timer                               | yes       |                                                                |
 |                               | UART                                | yes       | USART is shared with SPI. LEUART baud rate limited (see below) |
 |                               | USB                                 | no        |                                                                |
+| LCD driver                    | LS013B7DH0                          | &mdash;   |                                                                |
```

Comment:
LCD driver is supported using U8g2. Just like with the SLSTK3401A and SLSTK3402A.

Reply:
Okay. The board comes with the LCD though. Shouldn't we add the LCD driver to the set of implementations, even if it's not completed yet?

Reply:
The driver is completed, so it should work.